pkg/util/wait/wait_test.go

```diff
@@ -173,8 +173,25 @@ func TestJitterUntilNegativeFactor(t *testing.T) {
 }
 
 func TestExponentialBackoff(t *testing.T) {
+	sleeps := []time.Duration{}
+	sleepFn = func(d time.Duration) { sleeps = append(sleeps, d) }
```

Comment:
e.g., while this is a global this test can't be run in parallel with basically anything...